Instructions

It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. 

In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. 

Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

Set up the hypothesis test.

Write down all the steps followed for setting up the test.

Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.

If you finished the previous question, please go through the code for 
principal_component_analysis_example provided in the files_for_lab folder .

In [1]:
import math
from scipy import stats

H0: There is no significant difference in the mean blood pressure between the people in the Honolulu Heart Study and the regular population

In [2]:
n = 100
pop_mean = 120
sample_mean = 130.1
sample_std = 21.21

statistic = (sample_mean - pop_mean) / (sample_std / math.sqrt(n))

In [3]:
print(f' The test statistic is: ', statistic)

 The test statistic is:  4.761904761904759


At a 95% signicicance level (alpha = 0.05) we will reject the null hypothesis if the test statistic is below -1.98 or above 1.98

Since 4.76 > 1.98 we can reject the null hypothesis

Answer: At a 95% confidence level we make the assumption that there is a difference  in the mean blood pressure between the people in the Honolulu Heart Study and the regular population
